blueprints/addon/files/addon-config/ember-try.js

```diff
@@ -55,6 +55,19 @@ module.exports = {
           'ember': 'canary'
         }
       }
+    },
+    {
+      name: 'ember-source',
+      bower: {
+        dependencies: {
+          'ember': null
+        }
+      },
+      npm: {
+        dependencies: {
+          'ember-source': 'latest'
```

Comment:
Seems like this isn't needed? Since we already depend on the "right" version of ember-source in package.json...